decidim-admin/config/routes.rb

```diff
@@ -23,6 +23,7 @@
     resources :scopes, except: [:show] do
       resources :scopes, except: [:show]
     end
+    resources :statistics, only: [:index], controller: "statistics"
```

Comment:
```suggestion
    resources :statistics, only: [:index]
```
It might not be necessary to specify the controller

Reply:
Done